js/classes/entities/midground/chad.js

```diff
@@ -1,58 +1,392 @@
+        this.name = "dChad";
     };
 
-    /** The size, in pixels, of Chad's sprite ON THE SPRITESHEET. */
+    /** The size, in pixels of the sprite ON THE SPRITESHEET. */
     static get SIZE() {
+        return new Vector(29, 49);
+    }
 
-    };
-
-    /** How much bigger should Chad's sprite be drawn on the canvas than it is on the spritesheet? */
+    /** How much bigger should the sprite be drawn on the canvas than it is on the spritesheet? */
     static get SCALE() {
-
+        return 3;
     };
 
-    /** This will be Chad's size ON THE CANVAS. */
+    /** This will be the size of Papa Chad ON THE CANVAS. */
     static get SCALED_SIZE() {
         return Vector.multiply(Chad.SIZE, Chad.SCALE);
+    }
+
+    static get SPEED() {
+        return Chad.SCALE * 100;
     };
 
-    /** The filepath to Chad's spritesheet. */
+    /** The filepath to Papa Chad's spritesheet. */
     static get SPRITESHEET() {
-        return "./sprites/chad.js";
+        return "./sprites/parents.png";
     };
-    
-    /** Change what Chad is doing and where he is. */
+
+    /** The velocity of the first jump */
+    get FIRST_JUMP_VELOCITY() {
+        return -500
+    }
+    /** The velocity of the double jump. */
+    get SECOND_JUMP_VELOCITY() {
+        return -600;
+    }
+
+    /** The mulitiplier that allows CHAD to run. */
+    static get SPRINT_MULTIPLIER() {
+        return 0.6;
+    }
+    /** The mulitiplier that allows CHAD to run. */
+    static get DASH_MULTIPLIER() {
+        return 3.5;
+    }
+
+    /** The barrier that limits the length of the longest dash. */
+    get DASH_BARRIER() {
+        return 250;
+    }
+
+    /**
+     * Deals with movement in the x direction including walking, running and dashing.
+     * 
+     * @param {String} direction left or right.
+     * @param {Integer} xVelocity Chad's velocity in the x direction.
+     * @param {GAME} walkInDirection GAME.movingLeft or GAME.movingRight user actions.
+     * @param {GAME} running GAME.movingLeftShift, user intends for Chad to run.
+     * @param {GAME} dashingInEitherDirection GAME.running, user intends to dash mid-air.
+     * @param {Integer} toggleSign either a 1 or -1 to change the velocity direction.
+     * @returns xVelocity so that Chad's velocity can be updated.
+     * @author Caleb Krauter
+     */
+    manageXDirectionMovement(direction, xVelocity, walkInDirection, running, dashing) {
+        if (this.isOnGround) {
+            this.hasDoubleJumped = false;
+            this.canDash = true;
+            this.hasDashed = false;
+        }
+        const toggleSign = this.facing === "left" ? -1 : 1;
+
+        // Walk action
+        this.action = "walking";
+        this.facing = direction;
+        xVelocity += toggleSign * Chad.SPEED;
+
+        // Run action
+        if (running) {
+            this.action = "running";
+            this.facing = direction;
+            xVelocity += toggleSign * Chad.SPEED * Chad.SPRINT_MULTIPLIER;
```

Comment:
This makes running faster than it actually is. You have already added walk speed - you ought reset it to equal run speed instead of adding run speed to walk speed to get super speed.